release.sh

```diff
@@ -6,33 +6,60 @@
 # state.  The second phase is modifying files, tagging, commiting and pushing
 # to pypi
 
+OFFICIAL_GIT_REPO='git@github.com:taskcluster/taskcluster-client.py'
+
+# step into directory containing this script
+cd "$(dirname "${0}")"
+
+# exit in case of bad exit code
 set -e
 
 # Phase 1: Testing things
-if [ -z $VERSION ] ; then
-  echo "You must specify a version using \$VERSION"
-  exit 1
+
+# VERSION should have form x.y.z where x, y, z are positive integers
+if ! echo "${VERSION}" | grep -q '^\(0\|[1-9][0-9]*\)\.\(0\|[1-9][0-9]*\)\.\(0\|[1-9][0-9]*\)$'; then
```

Comment:
It is a messy regex, hey! The reason I used this form is that the only number that can start with 0 is 0 itself, i.e. we don't want a version like 04.02.30 but rather 4.2.30. So this means either a 0 by itself, or 1 of [1-9] followed by 0 or more of [0-9]. This way the only number that starts with a 0 is 0, and we also ensure that each number has at least one character (so the empty string does not match).